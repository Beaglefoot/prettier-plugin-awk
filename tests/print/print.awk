{
  print(a)
  print(a,b)
  print
  print $1
  print "hi" "bye"
  print "a","b"
}
