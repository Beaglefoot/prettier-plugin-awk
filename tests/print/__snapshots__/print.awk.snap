{
  print(a)
  print(a,b)
  print
  print $1
  print "hi" "bye"
  print "a","b"
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    print a
    print a, b
    print
    print $1
    print "hi" "bye"
    print "a", "b"
}
